src/app/components/home/PositionDetailPanel.tsx

```diff
@@ -131,6 +139,28 @@ const PositionDetailPanel = () => {
 
   const pos = useCollateralLockedSliderPos();
 
+  const openRebalancing = React.useCallback(() => setShowRebalancing(true), [setShowRebalancing]);
+  const closeRebalancing = React.useCallback(() => setShowRebalancing(false), [setShowRebalancing]);
+  const [shouldShowRebalancingAvaragePrice, setShouldShowRebalancingAvaragePrice] = React.useState(false);
+  const shouldShowSeperateTooltip = useMedia('(min-width: 1000px)');
```

Comment:
I've installed the tool, looks good, thanks :)
We can't, this is strictly bound to < 1000px screen size because of this change of the layout:
![tooltip](https://user-images.githubusercontent.com/13098459/144295055-4ebb0c35-b884-4823-9a98-d1687eac8bea.png)